cmd/vsphere-xcopy-volume-populator/internal/populator/vib.go

```diff
@@ -0,0 +1,75 @@
+package populator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kubev2v/forklift/cmd/vsphere-xcopy-volume-populator/internal/vmware"
+	"github.com/vmware/govmomi/object"
+	"k8s.io/klog/v2"
+)
+
+const (
+	vibName     = "vmkfstools-wrapper"
+	vibVersion  = "0.1.0"
```

Comment:
Could we reuse the `CUSTOM_VIB_VERSION` so we won't run out of sync in future?

Reply:
I thought about a build flag that I'll push using LDFLAG

Reply:
done, 
see vmkfstools-wrapper/version.mk usage in the vsphere-xcopy-volume-populator/Makefile and vsphere-xcopy-volume-popualtor/vmkfstools-wrapper/Makefile